app/src/main/java/ca/unb/mobiledev/reflexrevolution/GameActivity.java

```diff
@@ -0,0 +1,162 @@
+package ca.unb.mobiledev.reflexrevolution;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.CountDownTimer;
+import android.view.View;
+import android.widget.Button;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class GameActivity extends AppCompatActivity {
+    //Time in ms
+    //Could get rid of this and do random intervals or scale it off of score
+    private final int TIME_BETWEEN_LOOPS = 1000;
+
+    private TextView timeText;
+    private TextView scoreText;
+    private LinearLayout layout; //Layout we should add new UI elements to
+
+    private CountDownTimer timer;
+    private String currentInstruction = null;
+    private ArrayList<String> instructions;
+
+    private int timeCount;
+    private int score;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
We need to add the other lifecycle methods as well and make sure that both timers will start/stop/pause/restart at the appropriate time to prevent issues/unknown behaviour. 

Example:
Currently, if you start a game then use the device back button to go back to the main menu, the timer is still going in the background and you'll be redirected to the game over screen after a few seconds. Stoping the timers `onDestroy` should fix this particular case, I think.

Reply:
I'm not sure what you mean about the onDestroy thing... do you think it would work if I just paused the timer onPause() and then resumed it onResume()?

Reply:
Yeah so I don't really know the specific of how the countdown timer works so just pausing/resuming in the onPause() and onResume() might be enough. The onDestroy() method is called when the app is closed, so at that point the timer should definitely be stopped (I don't know if just pausing it might make it stay in memory and cause memory leaks, but I think that is unlikely)